### DISCUSSION TITLE
问一下，本项目默认gemini模型调用的是gemini pro是v1还是v1.5

### DISCUSSION THREAD

**Discussion by bestZwei** [ChatGPTNextWeb/NextChat#4309] [Q&A]
希望能做区分支持

**Comment by iliyian** [comment]
1.5的api还没完全开放吧

**Comment by RubuJam** [comment]
你混淆了两个东西，[API Version](https://ai.google.dev/docs/api_versions)只有V1和V1beta两个版本，目前项目使用的是V1beta，模型有gemini-1.0-pro和gemini-1.5-pro-latest等，目前默认使用的是gemini-pro，我在PR [#4412](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/pull/4412)中，提交了更新，规范模型名称的同时，支持不同的gemini模型，包括最新的gemini-1.5-pro-latest
